modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1148,6 +1148,84 @@ def _apply_func_to_list_of_partitions(cls, func, partitions, **kwargs):
         preprocessed_func = cls.preprocess_func(func)
         return [obj.apply(preprocessed_func, **kwargs) for obj in partitions]
 
+    @classmethod
+    def convert_partitions_to_pandas_in_remote_func(cls, partitions):
+        """
+        Call to_pandas for the given NumPy array of partitions.
+
+        For a NumPy array of partitions create a single partition
+        that has the data for all partitions.
+
+        Parameters
+        ----------
+        partitions : np.ndarray
+            The partitions which have to be converted to a single partition.
+
+        Returns
+        -------
+        PandasDataframePartition
+            A PandasDataframePartition object which holds the data of all partitions.
+        """
+        cls.finalize(partitions)
```

Comment:
Do we really need this? See my question about necessacity of the call `list_of_blocks`. Do we really need both (they do similar stuff)? Do we need only one call (either finalize or list_of_blocks)?

Reply:
Got it,  as draining call queue is done is list_of_blocks, I will remove this.